src/Menu/BemMenuWalker.php

```
@@ -66,13 +65,18 @@ public function __construct(string $cssClassPrefix)
 	 *
 	 * @return void Parent Display element
 	 */
-	public function display_element($element, &$children_elements, $max_depth, $depth = 0, $args, &$output) // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps, PEAR.Functions.ValidDefaultValue.NotAtEnd
-	{
-
+	public function display_element( // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps, PEAR.Functions.ValidDefaultValue.NotAtEnd
+		$element,
+		&$children_elements,
+		$max_depth, // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps, PEAR.Functions.ValidDefaultValue.NotAtEnd
+		$depth = 0,
+		$args, // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps, PEAR.Functions.ValidDefaultValue.NotAtEnd
+		&$output
```

Nothing wrong here, just curious, this is a PSR-12 thing or PHPStan thing or something? Like the line is too long so we need to break it up?

---

Too many arguments to fit in one line so I split it up. We can revert this and just ignore any errors.